boards/arm/lpcxpresso54114_m0/lpcxpresso54114_m0.yaml

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (c) 2017, NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+identifier: lpcxpresso54114
+name: NXP LPCXpresso54114
+type: mcu
+arch: arm
+ram: 64
+flash: 256
```

Comment:
should this be 64 since its pointing to sram1?

Reply:
Ok, fixed.